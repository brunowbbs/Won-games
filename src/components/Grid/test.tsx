import { Grid } from ".";
import { renderWithTheme } from "../../utils/tests/helpers";

describe("<Grid/>", () => {
  it("Should render correctly", () => {
    const { container } = renderWithTheme(<Grid>Children</Grid>);

    expect(container.firstChild).toMatchSnapshot();
  }); 
});
