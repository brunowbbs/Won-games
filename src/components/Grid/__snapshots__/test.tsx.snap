// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Grid/> Should render correctly 1`] = `
<div
  class="sc-bcXHqe hFXxTw"
>
  Children
</div>
`;
